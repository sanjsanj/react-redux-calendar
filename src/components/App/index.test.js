import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import { App } from ".";

describe("App", () => {
  it("should match snapshot", () => {
    const props = { showReminderModal: false };
    const wrapper = shallow(<App {...props} />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
