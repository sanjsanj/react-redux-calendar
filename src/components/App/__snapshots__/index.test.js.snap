// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<ThemeProvider
  theme={
    Object {
      "appBackgroundColor": "linear-gradient(to bottom,#292a49 0%,#2b548f 100%)",
      "appColor": "#fff",
      "boxShadowActive": "0px 3px 9px #000000",
      "boxShadowPassive": "0px 2px 9px #0000007a",
      "buttonBackground": "teal",
      "buttonColor": "#fff",
      "dayBackgroundLinearGradient": "linear-gradient(to bottom,#499bea 0%,#338ae7 100%)",
      "dayColor": "#000",
      "disabledDayBackgroundColor": "#0c4b8f",
      "mainTextColor": "#a7d1e2",
      "mainTitleColor": "#e0321f",
      "maxWidth": "700px",
      "modalUnderlayBackground": "#0000008a",
    }
  }
>
  <GlobalStyleComponent />
  <styled.h1>
    <img
      alt="Logo"
      src="favicon.ico"
    />
    Calendar Component
  </styled.h1>
  <ConnectFunction />
</ThemeProvider>
`;
